src/layup_cmdline/predict.py

```diff
@@ -11,13 +17,118 @@ def main():
         formatter_class=argparse.ArgumentDefaultsHelpFormatter,
         description="This would start predict",
     )
+
+    positionals = parser.add_argument_group("Positional arguments")
+
+    positionals.add_argument(
+        help="input orbit file",
+        dest="input",
+        type=str,
+    )
+
     optional = parser.add_argument_group("Optional arguments")
+
+    optional.add_argument(
+        "--ar",
+        "--ar-data-path",
+        help="Directory path where Assist+Rebound data files where stored when running bootstrap_sorcha_data_files from the command line.",
+        type=str,
+        dest="ar_data_file_path",
+        required=False,
+    )
+    optional.add_argument(
+        "-c",
+        "--conf",
+        help="optional configuration file",
+        type=str,
+        dest="c",
+        required=False,
+    )
     optional.add_argument(
-        "-p",
-        "--print",
-        help="Prints statement to terminal.",
-        dest="p",
+        "-ch",
+        "--chunksize",
+        help="number of orbits to be processed at once",
+        dest="chunk",
+        type=int,
+        default=10000,
+        required=False,
+    )
+
+    optional.add_argument(
+        "-d",
+        "--days",
+        help="number of days to predict on sky positions. Only used if the end date is not specified on the command line",
+        dest="days",
+        type=int,
+        default=14,
+        required=False,
+    )
+    optional.add_argument(
+        "-e",
+        "--end-date",
+        help="end date, mjd. Only specified if not specifying the number of days",
+        dest="e",
+        type=str,
+        default=None,
+        required=False,
+    )
+
+    optional.add_argument(
+        "-f",
+        "--force",
         action="store_true",
+        help="Overwrite output file",
+    )
+
+    optional.add_argument(
+        "-i",
+        "--input-type",
+        help="input format type of file (csv or hdf5)",
+        dest="i",
+        type=str,
+        default="csv",
+        required=False,
+    )
+
+    optional.add_argument(
+        "-n",
+        "--num-workers",
+        help="Number of CPU workers to use for parallel processing each chunk. -1 uses all available CPUs.",
+        dest="n",
+        type=int,
+        default=-1,
+        required=False,
+    )
+
+    optional.add_argument(
+        "-o",
+        "--output",
+        help="output file stem. default path is current working directory",
+        dest="o",
+        type=str,
+        default="predicted_output",
+        required=False,
+    )
+
+    optional.add_argument(
+        "-s",
+        "--start-date",
+        help="start date as jd_TDB or date string YYYY-mm-dd that will be converted to jd_TDB. If not specified default wil be current date",
+        dest="s",
+        type=str,
+        default=str(
+            Time(datetime.now(timezone.utc), format="datetime", scale="utc").tdb.jd
```

Comment:
@mschwamb Should the scale here also be "tdb"?

Reply:
because timezone is utc it's okay to be scale utc because then it's converted to tdb - but we could just go straight to tdb

Reply:
but then I think the timezone has to change as well if scale = tdb